### DISCUSSION TITLE
Download/Upload Time Calculator

### DISCUSSION THREAD

**Discussion by voodoopt** [CorentinTh/it-tools#1216] [Ideas]
Is it possible to do something like this?
https://www.omnicalculator.com/other/download-time
another little tool that might come in handy

**Comment by sharevb** [comment]
Hi @voodoopt , #1289  and available here : https://sharevb-it-tools.vercel.app/data-transfer-rate-converter and if you arr interested in up to date version of it-tools, I made kind of a fork here : https://github.com/sharevb/it-tools (https://sharevb-it-tools.vercel.app/ and docker images https://github.com/sharevb/it-tools/pkgs/container/it-tools)
